sympy/solvers/solvers.py

```diff
@@ -350,7 +350,11 @@ def check_assumptions(expr, against=None, **assumptions):
         if test is expected:
             continue
         elif test is not None:
-            return False
+            if get is True:
+                return False, key
```

Comment:
I have solved the issue of returning only one wrong assumption.
Now it is returning all the wrong assumptions
```
>>> check_assumptions(-5, True, integer=False, real=False, positive=True)
(False, ('integer', 'real', 'positive') )
```
But I am having difficulty in understanding the test cases.